lib/actions/enter.coffee

```diff
@@ -0,0 +1,75 @@
+###
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+getSubShellCommand = (command) ->
```

Comment:
This function is a perfect one to split it as an independent NPM module. I remember I used a previous module that ran the command using `cmd.exe` on Windows but defaulted to `sh` for Cygwin. Maybe we can create a new module that attempts to detect if the user is running on Cygwin, and use `sh` or `cmd.exe` accordingly?